src/math.rs

```diff
@@ -16,11 +16,59 @@ use cgmath::{
     BaseFloat, Decomposed, EuclideanSpace, InnerSpace, Point3, Quaternion, Rotation, Vector2,
     Vector3,
 };
-use collision::{Aabb, Aabb3};
+use collision::{Aabb, Aabb3, Contains};
 use num_traits::identities::One;
 use num_traits::Float;
+use std::convert::From;
+use std::fmt::Debug;
 use std::ops::Mul;
 
+pub trait PointCulling<S>: objekt::Clone + Debug + Sync + Send
+where
+    S: BaseFloat + Sync + Send,
+{
+    fn contains(&self, point: &Point3<S>) -> bool;
+    fn intersects(&self, aabb: &Aabb3<S>) -> bool;
```

Comment:
Could you add a TODO here to return Relation?

Reply:
done

Reply:
Nit: Our formatting convention for TODOs is:
```rust
// TODO(username): Do something.
```
Can you please adjust it everywhere in this PR?

Reply:
done

Reply:
Supernit: Still not in the format
```
// TODO(username): Do something.
```
everywhere.

Reply:
My bad, I just paid attention to the caps! now it should be finally fixed!